client-ui/src/app/components/lineage/lineage-details/operation-properties-details/property-error/property-error.component.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component } from '@angular/core';
+import { PropertiesComponent } from '../properties/properties.component';
+
+@Component({
+  selector: 'app-property-error',
+  templateUrl: './property-error.component.html',
+  styleUrls: ['property-error.component.less']
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [' should be "](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4600414)